helm/pinot/templates/broker/service-external.yaml

```diff
@@ -29,6 +29,7 @@ metadata:
     {{- include "pinot.brokerLabels" . | nindent 4 }}
 spec:
   type: {{ .Values.broker.external.type }}
+  publishNotReadyAddresses: true
```

Comment:
let's make default external to false here.

Reply:
done